### ISSUE TITLE
String literal concat

### ISSUE THREAD

**Issue by lt** [tqbf/jailbreak-trainer#3]
In the C editor string literal concatenation results in multiple PUSHARGs

i.e.

```C
field_unlock(
    "\xff...." // does stuff
    "\xff...." // other stuff
);
```

Results in 2 arguments being pushed instead of the literals being concatenated.

**Comment by tqbf** [comment]
Yep. String literal concat isn't supported yet. I'll post a list of what does/doesn't work in a few minutes.

(Leaving this open because I do plan to fix this.)
